src/emoji-font.css

```
@@ -1,45 +1,46 @@
+html.ytaf-legacy-emoji yt-formatted-string,
+html.ytaf-legacy-emoji yt-core-attributed-string,
+html.ytaf-legacy-emoji .yt-tv-text,
+html.ytaf-legacy-emoji .video-title,
+html.ytaf-legacy-emoji .title,
+html.ytaf-legacy-emoji #title,
+html.ytaf-legacy-emoji .description,
+html.ytaf-legacy-emoji #description,
+html.ytaf-legacy-emoji .video-title-text,
+html.ytaf-legacy-emoji .badge-text {
+  font-family:
+    'Roboto', 'YouTube Noto', 'YouTube Sans', 'Noto Sans Arabic', 'Arial',
```

Stylelint (17.14.0)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)